service/history/queue/action.go

```diff
@@ -50,13 +56,33 @@ type (
 	GetStateActionResult struct {
 		States []ProcessingQueueState
 	}
+
+	// GetTasksAttributes contains the parameter to get tasks
+	GetTasksAttributes struct{}
+	// GetTasksResult is the result for performing GetTasks Action
+	GetTasksResult struct {
+		tasks []task.Task
+	}
+	// UpdateTaskAttributes contains the parameter to update task
+	// TODO: replace result interface with defined data struct
+	UpdateTaskAttributes struct {
```

Comment:
- Since tasks will also be responded in batch, probably make this a batch operation as well?
- Update the type of result to `[]*types.CrossClusterTaskResponse` now that it has been defined? TaskID is probably not necessary as it's included in the TaskResponse type.

Reply:
updated